source/core/Console.cs

```diff
@@ -721,13 +727,30 @@ static unsafe void DisableControlsThisFrame()
 				NativeFunc.InvokeInternal(0x351220255D64C155 /*ENABLE_CONTROL_ACTION*/, 0, i, 0);
 		}
 
-		static unsafe float GetTextLength(string text)
+		static unsafe float GetTextLength(char* str, int count, float marginLength)
 		{
+			var calculated = count;
+			if (calculated > 50) { calculated = 50; }
```

Comment:
I wonder where this `50` comes from.

Reply:
Tried with 99, but the issue persists. So i just picked a random moderate number